package.json

```diff
@@ -5,7 +5,8 @@
   "scripts": {
     "lint": "eslint *.js bin lib tests",
     "test": "npm run-script lint && npm run-script utest",
-    "utest": "istanbul cover _mocha tests tests/ci tests/vcs"
+    "utest": "istanbul cover _mocha tests tests/ci tests/vcs",
+    "debug": "node $NODE_DEBUG_OPTION node_modules/.bin/_mocha tests tests/ci tests/vcs"
```

Comment:
You shouldn't need the `node_modules/.bin/` path, when `npm` runs something within a script, it looks there for binaries.